history/management/commands/run_bot.py

```diff
@@ -288,6 +289,10 @@ def won(message,opponentname,gamename):
             #send response
             gifurl = get_gif('winorloss')
             message.send("#win recorded \n {}".format(gifurl))
```

Comment:
not directly related to this PR, but im thinking of removing the gifs from win/loss records.  it just becomes too much to be in the channel with gifs every 2 lines.  as a user, what do you think?